openslides/utils/rest_api.py

```diff
@@ -155,14 +157,30 @@ def get_serializer_class(self) -> Type[Serializer]:
         Overridden method to return the serializer class given by the
         access permissions container.
         """
-        if self.get_access_permissions() is not None:
-            serializer_class = self.get_access_permissions().get_serializer_class(self.request.user)  # type: ignore
-        else:
+        model = self.get_queryset().model  # type: ignore
+        try:
+            return modelserializer[model]
+        except AttributeError:
             serializer_class = super().get_serializer_class()  # type: ignore
         return serializer_class
 
 
-class ModelSerializer(_ModelSerializer):
+modelserializer: Dict[Type[Model], Serializer] = {}
```

Comment:
Please rename this to something like `model_serializer_classes`

Reply:
done